app/models/order.rb

```diff
@@ -0,0 +1,4 @@
+class Order < ApplicationRecord
+  validates :user_id, presence: true
+  has_many :order_accessories
+end
```

Comment:
@agpiermarini Should an order belong_to a user?

Reply:
I'm pretty sure it does. How else are we going to display all orders for a particular user on their page?

Reply:
Yes, it will. But rather than add in associations wholesale, I thought it best to include what needed to be included to complete the story, and then add the others as we go.